bin/build.py

```diff
@@ -302,7 +304,7 @@
   os.chdir(build_dir)
   if args.test:
```

Comment:
You can test `if args.test or args.test_xml` so there will be no need to add `--test` every time you use `--test-xml`.